src/utils/common.js

```diff
@@ -128,7 +128,6 @@ export function prefsGet(...args) {
  * @return {Promise} 		prefs object which has been set, or error
  */
 export function prefsSet(prefs) {
-	log('PREFS SET', prefs);
```

Comment:
That's been there since 2018, are we sure we want to remove? `log` outputs in debug mode only (although looks like you removed it so it doesn't break the Jest setup file test?)

Reply:
sounds like Tino removed it, so since he's the one who put it there in the first place, nvm, heh